src/AddTodoForm.jsx

```diff
@@ -1,10 +1,15 @@
-import React from 'react';
+const AddTodoForm = (props) => {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const todoTitle = event.target.title.value;
+    props.onAddTodo(todoTitle);
```

Comment:
todoTitle variable is not really needed it, as it does not use in multiple places.